src/java/arjdbc/jdbc/RubyJdbcConnection.java

```diff
@@ -1791,7 +1791,7 @@ public static IRubyObject setRawDateTime(final IRubyObject self, final IRubyObje
     protected static IRubyObject typeCastFromDatabase(final ThreadContext context,
         final IRubyObject adapter, final RubySymbol typeName, final RubyString value) {
         final IRubyObject type = adapter.callMethod(context, "lookup_cast_type", typeName);
-        return type.callMethod(context, "type_cast_from_database", value);
+        return type.callMethod(context, "deserialize", value);
```

Comment:
@rdubya I assume this is a general Rails 5 change?  My msqlite changes did not hit this (from what I recall) so I don't know if this is common for other adapters but it is in RubyJdbcconnection.java.